mm2src/mm2_main/src/mm2_tests.rs

```diff
@@ -6789,6 +6789,54 @@ fn test_mm2_db_migration() {
     .unwrap();
 }
 
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn test_get_current_mtp() {
+    let passphrase = "cMhHM3PMpMrChygR4bLF7QsTdenhWpFrrmf2UezBG3eeFsz41rtL";
+    // KMD coin config used for this test
+    let coins = json!([
+        {"coin":"KMD","txversion":4,"overwintered":1,"txfee":10000,"protocol":{"type":"UTXO"}},
+    ]);
+    let mm = MarketMakerIt::start(
+        json! ({
+            "gui": "nogui",
+            "netid": 9000,
+            "dht": "on",  // Enable DHT without delay.
+            "passphrase": passphrase,
+            "coins": coins,
+            "rpc_password": "password",
+            "i_am_seed": true,
+        }),
+        "password".to_string(),
+        None,
+    )
+    .unwrap();
```

Comment:
Please check if you can replace it with the `Mm2TestConf`
https://github.com/KomodoPlatform/atomicDEX-API/blob/e48e0af1da7044e9a4fc49264868b13ca15afe28/mm2src/mm2_main/src/mm2_tests/orderbook_sync_tests.rs#L1357-L1358